### PR TITLE
Sign in and sign up forms

### PR DESCRIPTION
**Pull request by yukim17**
image
image

### CODE REVIEW

**Comment by yukim17** [comment]
**Created some simple dumb views for sign in and sign up**

File: SlimApp/SignIn/SignInView.swift [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+//  SlimApp
+//
+//  Created by Ekaterina Grishina on 24/01/23.
+//
+
+import SwiftUI
+
+struct SignInView: View {
+
+    @ObservedObject var viewModel: SignInViewModel
```
Comment (palant-dev) [inline SlimApp/SignIn/SignInView.swift]: Like Ahmed suggested try to instantiate the Model here

File: SlimApp/SignUp/SignUpView.swift [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+struct SignUpView: View {
+
+    @ObservedObject var viewModel: SignUpViewModel
+
+    var body: some View {
+        VStack {
+            Text("Create an account")
+                .font(.title)
+                .padding(.bottom, 64)
+            TextField("", text: $viewModel.email, prompt: Text("Email"))
```
Comment (palant-dev) [inline SlimApp/SignUp/SignUpView.swift]: Maybe using the label without prompt is better in order to have less problems in the future with Dark Mode

**Review by palant-dev — COMMENTED** [review]
Fix proposed: 
- Instantiate the model inside SigniInView
- Label instead of prompt for TextViews

File: SlimApp/SignUp/SignUpView.swift (hunk 2 above)
Comment (yukim17) [reply SlimApp/SignUp/SignUpView.swift]: I've changed a bit colors for dark mode, so now I need to use prompt in order placeholder to be brighter in dark mode

Comment (yukim17) [reply SlimApp/SignUp/SignUpView.swift]: image

**Pull request merged**